js/layout-javascript/agenda-code.js

```diff
@@ -989,6 +989,7 @@ DynamicList.prototype.initialize = function() {
     Fliplet.App.Storage.remove('flDynamicListQuery:' + _this.data.layout);
   }
 
+  moment.locale(navigator.language);
```

Comment:
You removed a line of `moment.locale('en')` in `utils.js`. Why didn't you just replace that line with this line here, instead of having to repeat this in every layout JS file?